e2e/tests/features/sdk_website.feature

```diff
@@ -0,0 +1,29 @@
+Feature: SDK Demo Site
+  As an end user,
+  I want to use SDK Demo site to check if TDF3 files worked,
+  So I can develop my own app using TDF3.
+
+  # Scenarios can be highly declarative, which focuses on behavior.
+  # Don't get caught up in button names and layouts at the Gherkin level.
+  # Note that these scenarios use Selenium WebDriver to do browser interactions.
+
+  # for every Then assertion, we require a case_id, which is TestRail Case ID
+  # https://virtru.testrail.net/index.php?/dashboard
+  # Talk to QA Leads before you create Test Cases in TestRail
+
+  Background:
+    Given Project is protect-and-track-demo
+
+  @regression
+  Scenario Outline: Decrypt TDF3 file in SR
```

Comment:
IIUC this is kinda unrelated to the repository here, right? Maybe add a TODO to describe what we should replace it with?